### PR TITLE
Add deployment & delete commands

### PR DESCRIPTION
**Pull request by Avielyo10**

### CODE REVIEW

File: prometheus/cli.py [hunk 1]
```diff
@@ -71,3 +73,36 @@ def metrics(host, token, interval, time, skip_namespaces, output, sort_by):
...
+def get_data_file_dir():
+    files_path = os.path.join(sys.exec_prefix, 'local', '.prom')
+    if not os.path.isdir(files_path):
+        files_path = os.path.join(sys.exec_prefix, '.prom')
+        if not os.path.isdir(files_path):
+            raise Exception("[ERROR] Couldn't find data files")
+    list_of_files = ["prometheusRules.yaml", "configmap.yaml",
+                     "service.yaml", "servicemonitor.yaml", "daemonset.yaml"]
+
+    return files_path, list_of_files
```
Comment (Avielyo10) [inline prometheus/cli.py]: @lack IDK if that's the right place for it, want to know what do you think

Comment (lack) [reply prometheus/cli.py]: Can't we just glob for `*.yaml` and install all of those?  Or concatenate then into a single yaml?

Comment (Avielyo10) [reply prometheus/cli.py]: Yes we can, I believe this will be smarter

**Review by lack — CHANGES_REQUESTED** [review]
Looks good! 

But we should update README.md with the new usage info too.

**Review by lack — APPROVED** [review]

**Pull request merged**